libs/full/async_distributed/CMakeLists.txt

```diff
@@ -120,7 +120,8 @@ add_hpx_module(
   HEADERS ${async_headers}
   COMPAT_HEADERS ${async_compat_headers}
   DEPENDENCIES hpx_core hpx_parallelism
-  MODULE_DEPENDENCIES hpx_actions hpx_naming hpx_runtime_configuration
-                      hpx_runtime_local hpx_threadmanager
+  MODULE_DEPENDENCIES
+    hpx_actions hpx_naming hpx_runtime_configuration hpx_runtime_local
+    hpx_serializable_function hpx_threadmanager
```

Comment:
`hpx_serializable_function` does not need to (should not) be here. It comes indirectly through `hpx_core`, the shared library which `hpx_serializable_function` is part of.